julia/core.py

```diff
@@ -30,6 +30,8 @@
 # this is python 3.3 specific
 from types import ModuleType, FunctionType
 
+from setuptools.pep425tags import get_abi_tag
```

Comment:
...and it fails in Windows because they don't have setuptools by default? https://ci.appveyor.com/project/Keno/pyjulia/build/1.0.98/job/kh23o578ngvjkysy#L89